### PR TITLE
Add plugin:prettier/recommended

### PR DESCRIPTION
**Pull request by azz**
Just added a `"recommended"` configuration to `eslint-plugin-prettier` in https://github.com/prettier/eslint-plugin-prettier/issues/72. Turning it on in this repo and documenting it here, too.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -5,8 +5,12 @@ module.exports = {
...
     node: true
   },
-  extends: ["eslint:recommended", "plugin:react/recommended"],
-  plugins: ["prettier", "react", "import"],
+  extends: [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended"
+  ],
+  plugins: ["import"],
```
Comment (azz) [inline .eslintrc.js]: removed `"react"` because it is turned on by `"plugin:react/recommended"`.

File: .eslintrc.js [hunk 2]
```diff
@@ -22,15 +26,9 @@ module.exports = {
     "one-var": ["error", "never"],
     "prefer-arrow-callback": "error",
     "prefer-const": "error",
-    "prettier/prettier": "error",
     "react/no-deprecated": "off",
     strict: "error",
     "symbol-description": "error",
     yoda: ["error", "never", { exceptRange: true }]
-  },
-  parserOptions: {
```
Comment (azz) [inline .eslintrc.js]: This is done by `eslint-plugin-react`.

File: docs/eslint.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ yarn add --dev prettier eslint-plugin-prettier
 
 ```json
 {
-  "plugins": ["prettier"],
+  "extends": ["plugin:prettier/recommended"],
   "rules": {
     "prettier/prettier": "error"
```
Comment (SimenB) [inline docs/eslint.md]: remove this as well?

Comment (azz) [reply docs/eslint.md]: Oops, didn't mean to change that block at all.

**Pull request merged**